### PR TITLE
NN-5154: use IRSA to connect to SQS and SNS - dev

### PR DESCRIPTION
**Pull request by neilmendum**
https://dsdmoj.atlassian.net/browse/NN-5154

### CODE REVIEW

File: helm_deploy/hmpps-manage-adjudications-api/values.yaml [hunk 1]
```diff
@@ -52,22 +52,6 @@ generic-service:
       DATABASE_NAME: "database_name"
       DATABASE_USERNAME: "database_username"
       DATABASE_PASSWORD: "database_password"
-    hmpps-domain-events-topic:
```
Comment (neilmendum) [inline helm_deploy/hmpps-manage-adjudications-api/values.yaml:55]: Moved into each environment's config, will revert once we're happy to make same changes in PreProd & Prod

File: helm_deploy/values-dev.yaml [hunk 2]
```diff
@@ -7,14 +7,27 @@ generic-service:
...
     host: manage-adjudications-api-dev.hmpps.service.justice.gov.uk
 
+  serviceAccountName: hmpps-manage-adjudications-api-dev
+
   env:
     APPLICATIONINSIGHTS_CONFIGURATION_FILE: applicationinsights.dev.json
     OAUTH_ENDPOINT_URL: https://sign-in-dev.hmpps.service.justice.gov.uk/auth
     SPRING_SECURITY_OAUTH2_RESOURCESERVER_JWT_JWK_SET_URI: https://sign-in-dev.hmpps.service.justice.gov.uk/auth/.well-known/jwks.json
     PRISON_API_ENDPOINT_URL: https://api-dev.prison.service.justice.gov.uk
+    HMPPS_SQS_USE_WEB_TOKEN: "true"
```
Comment (neilmendum) [inline helm_deploy/values-dev.yaml:17]: Added with https://github.com/ministryofjustice/hmpps-spring-boot-sqs/pull/138

File: helm_deploy/values-dev.yaml [hunk 3]
```diff
@@ -7,14 +7,27 @@ generic-service:
   ingress:
     host: manage-adjudications-api-dev.hmpps.service.justice.gov.uk
 
+  serviceAccountName: hmpps-manage-adjudications-api-dev
```
Comment (neilmendum) [inline helm_deploy/values-dev.yaml:10]: Created with https://github.com/ministryofjustice/cloud-platform-environments/pull/13564/files#diff-bb5e49105d80cf16080523042d632050e00278a8ce9112f1524573a487de964fR18

File: helm_deploy/values-dev.yaml [hunk 4]
```diff
@@ -7,14 +7,27 @@ generic-service:
...
     OAUTH_ENDPOINT_URL: https://sign-in-dev.hmpps.service.justice.gov.uk/auth
     SPRING_SECURITY_OAUTH2_RESOURCESERVER_JWT_JWK_SET_URI: https://sign-in-dev.hmpps.service.justice.gov.uk/auth/.well-known/jwks.json
     PRISON_API_ENDPOINT_URL: https://api-dev.prison.service.justice.gov.uk
+    HMPPS_SQS_USE_WEB_TOKEN: "true"
 
   allowlist: null
 
+  namespace_secrets:
+    hmpps-domain-events-topic:
+      HMPPS_SQS_TOPICS_DOMAINEVENTS_ARN: "topic_arn"
```
Comment (neilmendum) [inline helm_deploy/values-dev.yaml:23]: No longer need to provide `ACCESS_KEY_ID` or `SECRET_ACCESS_KEY` secrets as this is handled by IRSA

**Review by timoj58 — APPROVED** [review]

**Pull request merged**